server/temp/batch.conf

```diff
@@ -31,6 +31,9 @@ backend {
         // Base bucket for workflow executions
         root = "gs://cromwell-29292/cromwell-execution"
 
+        // docker's entry point
+        job-shell = "/bin/sh"
```

Comment:
This used to be hardcoded at `GcpBatchRequestFactoryImpl`

Reply:
That's already done but defaults to `/bin/bash`: https://github.com/Burwood/cromwell/blob/95f59cd31bb1fb699445f9f1c83bde21f0979be5/supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/GcpBatchConfiguration.scala#L37